airbyte-integrations/bases/base-java/src/main/resources/ssh-tunnel-spec.json

```diff
@@ -59,6 +59,7 @@
           "type": "string",
           "airbyte_secret": true,
           "multiline": true,
+          "pattern": "-----BEGIN RSA PRIVATE KEY-----(.|\n)*-----END RSA PRIVATE KEY-----",
```

Comment:
a valid key may also be "-----BEGIN OPENSSH PRIVATE KEY-----"
I'd put the expression `BEGIN` and `END` as any alphanumerical. 
Same for `END`